AUTHORS.md

```diff
@@ -0,0 +1,32 @@
+# Credits
+
+---
+© 2024 **xdem developers**.
+
+**xdem** is licensed under permissive Apache 2 license.
+
+All contributors listed in this document are part of the **xdem developers**, and their
+contributions are subject to the project's copyright under the terms of the
+[Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0).
+
+This file keeps track of authors contributions.
+
+## Contributors
+
+---
+
+Contributors are developers, researchers, or users who have submitted features, fixed bugs, or helped improve the
+project.
+
+- **Romain Hugonnet** [@rhugonnet](https://github.com/rhugonnet) <hugonnet@uw.edu>
+- **Amaury Dehecq** [@adehecq](https://github/adehecq) <amaury.dehecq@univ-grenoble-alpes.fr>
+- **Erik Schytt Mannerfelt** [@erikmannerfelt](https://github/erikmannerfelt)
+- **Friedrich Knuth** [@friedrichknuth](https://github/friedrichknuth)
+- **Andrew Tedstone** [@atedstone](https://github/atedstone)
+- **Zhihao LIU** [@liuh886](https://github/liuh886)
+- **Diego Cusicanqui** [@cusicand](https://github/cusicand)
+- **Alessandro Gentilini** [@alessandro-gentilini](https://github/alessandro-gentilini)
+- **Ferdinand Schenck** [@fnands](https://github/fnands)
+- **Johannes Landmann** [@jlandmann](https://github/jlandmann)
+
+Update here with new contributors.
```

Comment:
Could you redesign this file to make it look more like this one https://github.com/CNES/Pandora2D/blob/master/AUTHORS.md?